app/flux/components/CategoryListItem.react.js

```diff
@@ -0,0 +1,43 @@
+var { Link } = require('react-router');
```

Comment:
Seems really weird to mix AMD with imports syntax, if you're going to use it go all the way:
````
import {Link} from 'react-router';
````